aria-practices.html

```diff
@@ -140,8 +140,8 @@ <h3>Browser and Assistive Technology Support</h3>
         It is thus advisable to test implementations thoroughly with each browser and assistive technology combination that is relevant within a target audience.
       </p>
       <p>
-        Similarly, JavaScript and CSS in this guide is written to be compatible with the most recent version of Chrome, Firefox, Internet Explorer, and Safari at the time of writing.
-        In particular, some JavaScript and CSS may not function correctly in Internet Explorer version 10 or earlier.
+        Similarly, JavaScript and CSS in this guide is written to be compatible with the most recent version of Chrome, Firefox, and Safari at the time of writing.
+        In particular, some JavaScript and CSS may not function correctly in Internet Explorer.
```

Comment:
I think we can nix the "In particular, " ...

```suggestion
Some JavaScript and CSS may not function correctly in Internet Explorer.
```